helm/aad-pod-identity-chart/values.yaml

```diff
@@ -0,0 +1,34 @@
+# Default values for pod-aad-identity-helm.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+rbac:
+  enabled: false
+
+azureIdentity:
+  name: "azure-identity"
+  type: 0 # type 0: MSI, type 1: Service Principal
+  resourceID: "/subscriptions/subscription-id/resourcegroups/resource-group/providers/Microsoft.ManagedIdentity/userAssignedIdentities/identity-name"
+  clientID: "identity-client-id"
```

Comment:
Let's make both `resourceID` and `clientID` here empty with comments as examples